core/src/scene/light.h

```diff
@@ -4,20 +4,21 @@
 #include "glm/glm.hpp"
 #include "util/shaderProgram.h"
 
-typedef enum {
-    DEFAULT,
-    VERTEX,
-    FRAGMENT
-} InjectionType;
-
 enum class LightType {
     UNDEFINED,
+    AMBIENT,
     DIRECTIONAL,
     POINT,
     SPOT,
     CUSTOM
 };
 
+enum class LightOrigin {
+    CAMERA,
+    GROUND,
+    WORLD
```

Comment:
GROUND is the camera position (in XY) projected on the floor with Z is on the ground
WORLD is absolute world position

More info at https://github.com/tangrams/tangram/wiki/Lights-Overview#origin